src/YesSql.Core/Sql/SchemaBuilder.cs

```diff
@@ -56,11 +56,16 @@ public ISchemaBuilder CreateMapIndexTable(Type indexType, Action<ICreateTableCom
 
                 createTable
                     .Column<int>("Id", column => column.PrimaryKey().Identity().NotNull())
-                    .Column<int>("DocumentId");
+                    .Column<int>("DocumentId")
+                    ;
 
                 table(createTable);
                 Execute(_commandInterpreter.CreateSql(createTable));
 
+                AlterTable(indexTable, table =>
+                    table.CreateIndex($"IDX_PK_{indexTable}", "Id")
```

Comment:
I thought it was the foreign keys that weren't getting the indexes?

Primary key always gets one, but the `CreateForeignKey` below is the one that doesn't get an index on `DocumentId`, which is the value used for the join back to `Document`

Reply:
I am glad I asked you to review, lame

Reply:
Updated

Reply:
Yes, as we removed the `CreateIndex()` for the `Identifiers` table through #350